bus/bus.go

```diff
@@ -497,6 +492,8 @@ func (b *Bus) Handler() http.Handler {
 		"POST   /syncer/connect": b.syncerConnectHandler,
 		"GET    /syncer/peers":   b.syncerPeersHandler,
 
+		"POST /system/database/backup": b.postSystemSQLite3BackupHandler,
```

Comment:
`hostd` calls this `/system/sqlite3/backup` which also works but it it feels a bit odd to implement. 
The bus currently has a nice `Store` interface which hides away the actual database. Since that is generic I called it `database` for now while returning an error if the underlying database doesn't support backups.

Reply:
Separating the endpoints allows for database specific backup flows. For something like MySQL or Postgres the endpoint might backup to another running server rather than a file.